diff --git a/node_modules/react-native-background-task/android/build.gradle b/node_modules/react-native-background-task/android/build.gradle
index 35c3988..9d5e380 100644
--- a/node_modules/react-native-background-task/android/build.gradle
+++ b/node_modules/react-native-background-task/android/build.gradle
@@ -1,22 +1,22 @@
 buildscript {
     repositories {
-        jcenter()
+        google()
+        mavenCentral()
     }
-
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.1.3'
+        classpath 'com.android.tools.build:gradle:7.4.2'
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 35
+    buildToolsVersion "35.0.0"
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 22
+        minSdkVersion 24
+        targetSdkVersion 35
         versionCode 1
         versionName "1.0"
     }
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
-    compile 'com.evernote:android-job:1.1.11'
+    implementation 'com.facebook.react:react-native:+'
+    implementation 'com.evernote:android-job:1.1.11'
 }
\ No newline at end of file
